js/scripts.js

```diff
@@ -120,28 +120,34 @@ function updateTargetLanguages() {
     }
 }
 
-function sendFeedback(feedback) {
+async function sendFeedback(feedback) {
     if ($('#sourceData').val() == '' || $('#translatedData').val() == '') {
         return;
     }
-    var params = {
-        TableName: 'TranslationFeedback',
-        Item: {
-            Timestamp: Date.now(),
-            SourceText: $('#sourceData').val(),
-            LanguageFrom: $('#languageFrom').val(),
-            LanguageTo: $('#languageTo').val(),
-            TranslatedText: $('#translatedData').val(),
-            Feedback: feedback
-        }
-    };
-    docClient.put(params, function (err, data) {
-        if (err) {
-            console.log(JSON.stringify(err));
-        } else {
-            console.log(JSON.stringify(data));
-        }
-    });
+
+            var Timestamp = Date.now();
+            var SourceText = document.getElementById('sourceData').value;
+            var LanguageFrom = document.getElementById('languageFrom').value;
+            var LanguageTo = document.getElementById('languageTo').value;
+            var TranslatedText = document.getElementById('translatedData').value;
+            var Feedback = feedback;
```

Comment:
Looks like this variable called "Feedback" (the one with a capital f) is redundant. When sending the data to the API, you end up using the one with lowercase f.

Reply:
Resolved